node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageSender.scala

```diff
@@ -99,6 +98,8 @@ case class PeerMessageSender(
            { case msgs: Vector[NetworkMessage] =>
              s"received msgs=${msgs.map(_.payload.commandName)} from peer=$peer"
            })
+      .withAttributes(Attributes.logLevels(onElement = Logging.DebugLevel,
```

Comment:
from: https://doc.akka.io/docs/akka/current/stream/stream-cookbook.html#logging-in-streams